lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -87,11 +87,25 @@
   // Implementation detail of `apos.utils.post` and `apos.utils.get`
   function monitor(xmlhttp, callback) {
     xmlhttp.addEventListener("load", function() {
+      var responseHeader = this.getResponseHeader("Content-Type");
+      var trimResponse = this.responseText.trim();
       var data;
-      try {
-        data = JSON.parse(this.responseText);
-      } catch (e) {
-        return callback(e);
+      if (responseHeader === 'application/json; charset=utf-8') {
```

Comment:
Hmm, just make sure it either equals `application/json` in its entirety, or starts with `application/json;`. I say this because in this context `charset` is supposedly a cargo cult thing that has no meaning (JSON must be utf-8) and therefore might not be present. It took some research to figure that out just now. Weird.